docs/patterns/async-request-reply.md

```diff
@@ -82,7 +82,7 @@ The following diagram shows a typical flow:
 ![Request and response flow for asynchronous HTTP requests](./_images/async-request.png)
 
 1. The client sends a request and receives an HTTP 202 (Accepted) response.
-2. The client sends an HTTP GET request to the status endpoint. The work is still pending, so this call also returns HTTP 202.
+2. The client sends an HTTP GET request to the status endpoint. The work is still pending, so this call also returns HTTP 200.
```

Comment:
What is this change based on?

Furthermore, the way it's written now is kinda weird "also returns HTTP 200" -- also implies that it did before.

Reply:
Hi @ckittel - the change is based on the previous change where the text and image were modified to say that the status endpoint returns `200` rather than `202`. This small bit of text was still referring to `202`.

Reply:
Ah. We'll the code shown (and even described) on this same page (and n GitHub) has 202 being returned. Maybe the right fix is to update the other places that say 200 instead -- that may have been a mistake to change prior? We really don't want the words to be mismatched with the implementation.

Alternatively, of 200 is the idiomatic response for this situation from a status endpoint, then the code should be updated as well (here and in GitHub) to complete this change.

Reply:
I think we change all instances to 202, unless something changed in the implementation?